deployment/terraform/create.go

```diff
@@ -994,7 +1000,10 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 	cfg.EmailSettings.SMTPServer = model.NewPointer(t.output.MetricsServer.PrivateIP)
 	cfg.EmailSettings.SMTPPort = model.NewPointer("2500")
 
-	if t.output.HasProxy() && t.output.HasS3Key() && t.output.HasS3Bucket() {
+	if t.output.HasEFS() {
```

Comment:
This will deploy EFS even on a single server installation. Is that intended?

Reply:
Yes, it should be doable and make sense.